src/Blocks/BlockSnow.h

```diff
@@ -93,6 +98,16 @@ class cBlockSnowHandler :
 	{
 		return a_Position.y < (cBlockInfo::GetBlockHeight(a_BlockType) * (a_BlockMeta & 7));
 	}
+
+private:
+
+	/** Returns true if snow can be placed on top of a block with the given type and meta. */
+	static bool CanBeOn(BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta)
+	{
+		// If block below is snowable, or it is a thin slow block and has a meta of 7 (full thin snow block), say yay
+		return (cBlockInfo::IsSnowable(a_BlockType) || ((a_BlockType == E_BLOCK_SNOW) && (a_BlockMeta == 7)));
```

Comment:
I know this is just copied from above, but preferably we don't want magic numbers. Any chance the meta's could have their own static value or is that overkill?

Reply:
The value is used in a few different places so I've added a constant `FullBlockMeta` for it.